PrebidMobile/Logging/Log.swift

```diff
@@ -33,38 +33,70 @@ public class Log: NSObject {
     public static var logToFile = false
 
     public static func error(_ object: Any, filename: String = #file, line: Int = #line, function: String = #function) {
-        log(object, logLevel: .error, filename: filename, line: line, function: function)
+        if (customLogger == nil) {
```

Comment:
In my opinion, this kind of functionality should be implemented in the following way: 

1. Introduce the class `SDKConsoleLogger` which implements `PrebidLogger` protocol
2. Introduce internal property `var logger:  PrebidLogger? = SDKConsoleLogger()`
3. Introduce a custom setter for the `logger` so pubs can set up own one. 
4. in all functions of the `Log` class just use `logger?.dosomething( .... )`

It will alow to avoid `if ... else` and seamsly add another SDK loggers like `SDKFileLogger` if needed.